tools/buildsys/src/builder.rs

```diff
@@ -367,8 +459,10 @@ impl DockerBuild {
             &self.state_dir,
         )?;
 
-        // Clean up any previous outputs we have tracked.
-        clean_build_files(&marker_dir, &self.artifacts_dir)?;
+        // Clean up any previous outputs we have tracked, unless we are repacking.
+        if self.target != "repack" {
+            clean_build_files(&marker_dir, &self.artifacts_dir)?;
+        }
```

Comment:
I'm not sure if I should skip this when repacking, or if it would be more correct to do something with the markers instead. The conditional was added because I noticed if you run `repack-variant` a second time the images disappear prior to `img2img` and the repackaging fails.